dev/add-delete-pas-application.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+# This script will create an application named $APP_NAME.
+# This application will only be able to authenticate from machine '$APP_ADDRESS' using the linux OS user '$APP_OS_USER'
+# This application will have retrieve access to all accounts inside of the '$SAFE_NAME' safe
+# In this example $APP_NAME application will have access to the MYSQL account specified below in 'Account Information'
+
+# Application information
+APP_NAME="FirstApplication"
+APP_OS_USER="firstapp"
+APP_ADDRESS="10.0.1.10"
+SAFE_NAME="TEST_${APP_NAME}"
+
+# Account information
+DATABASE_ACCOUNT_NAME="mysql"
+DATABASE_ADDRESS="10.0.1.12"
+DATABASE_USERNAME="firstapp"
+DATABASE_DEFAULT_PASSWORD="thisIsTheDefaultPassword"
+DATABASE_PLATFORM="MySQL"
+
+cybr logon -a ldap -b "$PAS_HOSTNAME" -u "$PAS_USERNAME"
```

Comment:
Every variable below this line should be double-quoted to prevent globbing as this line of code is.

Reply:
Resolved in recent commit